index.php

```diff
@@ -483,6 +101,11 @@
         //Ban info
         require 'templates/ban_info.php';
 
+        // Load user record
+        $route = Router::$routes[34];
```

Comment:
Will do. Thinking this will likely move to character select/user settings or something, since we can just restrict selecting/changing characters with game bans.